.travis.yml

```diff
@@ -23,6 +23,9 @@ matrix:
   - python: 3.6
     env: TOX_ENV=py36
 
+  - python: 3.6
+    env: TOX_ENV=isort
```

Comment:
can we call this check-isort, to stop me thinking it's going to somehow magically do our isorting?